ultrasonic/src/main/res/layout/select_folder_header.xml

```diff
@@ -24,7 +24,7 @@
         a:orientation="vertical" >
 
         <TextView
-            a:id="@+id/select_artist_folder_1"
+            a:id="@+id/select_folder_1"
```

Comment:
I know this was bad before, but I think if you change the name anyways, it can be renamed to something meaningful e.g. select_folder_title and select_folder_name below or something similar.